mappings/net/minecraft/container/GenericContainer.mapping

```diff
@@ -4,5 +4,21 @@ CLASS awz net/minecraft/container/GenericContainer
 	METHOD <init> (Laxt;ILaum;Lagk;I)V
 		ARG 2 syncId
 		ARG 5 rows
+	METHOD a Generic9x1 (ILaum;)Lawz;
```

Comment:
Should these not be camelCased?

Reply:
Yeah, they used to be classes before so the uppercase name made sense but now their methods so I'll change it.